### PR TITLE
add list of projects using prettier

### PR DESCRIPTION
**Pull request by codyzu**
As discussed in #1351 

This PR adds a list of "big name" projects using Prettier. I did not do any of the verification, rather, I simply put together the images and links in the readme.

A few notes:

The logos were taken from these (seemingly official) locations:
* [React](https://github.com/facebook/react/tree/master/docs/img)
* [Jest](https://github.com/facebook/jest/tree/master/website/src/jest/img)
* [Yarn](https://github.com/yarnpkg/website/tree/master/assets)
* [Babel](https://github.com/babel/logo)
* [Zeit](https://zeit.co/logos)
* [Webpack](https://webpack.js.org/branding/)

Afterwards, I converted svgs to pngs (to render on github) and I scaled the logos to have a max height of 100px. The Jest logo was white, so I reversed the colors (now black) to show up on the rendered markdown.

There is a link back to the issue to as suggested by @leipert.

I also took the liberty to add a link to the npm "depends on prettier" page.

Final note, I used the html tags (rather than markdown syntax) to remove the required table header.

@vjeux what do you think?

### CODE REVIEW

**Comment by vjeux** [comment]
Looks awesome!

Few comments:
- could you add a fixed width such that it doesn't jump when loading
- can you fix up the alignment and sizes, it looks off.

Can't wait to ship it.

File: README.md [hunk 1]
```diff
@@ -49,6 +49,21 @@ If you are interested in the details, you can watch those two conference talks:
...
+<table>
+<tr>
+<td><p align="center"><a href="https://facebook.github.io/react/"><img src="images/react.png"><br>React</a></p></td>
+<td><p align="center"><a href="https://facebook.github.io/jest/"><img src="images/jest.png"><br>Jest</a></p></td>
+<td><p align="center"><a href="https://yarnpkg.com"><img src="images/yarn.png"></a></p></td>
+</tr>
+<tr>
+<td><p align="center"><a href="https://babeljs.io/"><img src="images/babel.png"></a></p></td>
+<td><p align="center"><a href="https://zeit.co/"><img src="images/zeit.png"><br>Zeit</a></p></td>
+<td><p align="center"><a href="https://webpack.js.org/"><img src="images/webpack.png"></a></p></td>
```
Comment (leipert) [inline README.md]: Maybe link to webpack-cli?

**Comment by leipert** [comment]
Awesome 👍 
- But we probably want to put the images into `.npmignore` 
- Use full urls for the image links? (I do not know, how npm does handle relative image links)

**Comment by vjeux** [comment]
prettier uses a custom process to do the release so you don't need to worry about npm ignore.

File: README.md (hunk 1 above)
Comment (vjeux) [inline README.md]: Sounds good

Comment (codyzu) [inline README.md]: @leipert I changed it to webpack-cli

**Comment by codyzu** [comment]
> * could you add a fixed width such that it doesn't jump when loading
> * can you fix up the alignment and sizes, it looks off.

@vjeux I resized all the logos to the same size and set the width and height in the img tags. After that, I'm not sure what else I can do. But I think it makes it load more smoothly and the logos are consistent now.

> Use full urls for the image links? (I do not know, how the npm info page does handles relative image links)

>. Maybe link to webpack-cli, as the main webpack is not (yet?) using it

@leipert I can confirm from my own packages, npm does some magic to load images from relative paths. So that should not be a problem for npm. Therefore, IMO, it is best to leave them relative for durability. Also I changed webpack to webpack-cli.

Anything else I should do? Keep me posted :-) I'm happy to contribute to your great project!

**Comment by vjeux** [comment]
Thanks!

**Pull request merged**